roiextractors/segmentationextractor.py

```diff
@@ -205,6 +214,70 @@ def get_image_size(self) -> ArrayType:
         """
         pass
 
+    def get_traces(self, roi_ids=None, start_frame=None, end_frame=None, name='Fluorescence'):
+        """
+        Return RoiResponseSeries
+        Returns
+        -------
+        traces: array_like
+            2-D array (ROI x timepoints)
+        """
+        if roi_ids is None:
+            roi_idx_ = range(self.get_num_rois())
+        else:
+            roi_idx = [np.where(np.array(i) == self.roi_ids)[0] for i in roi_ids]
+            ele = [i for i, j in enumerate(roi_idx) if j.size == 0]
+            roi_idx_ = [j[0] for i, j in enumerate(roi_idx) if i not in ele]
+        traces = [getattr(self, i) for i in self.__dict__.keys() if name.lower() in i]
```

Comment:
I had a discussion with allessio and we decided to use the roi_response_dict to retrieve the traces instead. So this would line would go altogether. 
There are some variable naming specific changes we discussed like:
Making plural to singular, removing the property methods as they are redundant (only keeping the get methods). I was thinking to have that in another PR after this merge

Reply:
I'd rather not merge this line

Reply:
@bendichter 

This will be changed in a near future PR that is almost ready. Basically, we decided to have 4 internal attributes, to reflect the 4 main common responses that can be available:

- self._roi_response_raw: raw fluorescence trace 
- self._roi_response_dff: df over f fluorescence trace 
- self._roi_response_neuropil: neuropil fluorescence trace 
- self._roi_response_deconvolved: deconvolved trace 

In addition, there will be a `get_traces_dict()` method that packs them nicely, so the actual implementation of the get traces will be something like:

```
def get_traces(self, roi_ids=None, start_frame=None, end_frame=None, name='raw'):
        return get_traces_dict()[name][roi_ids, start_frame:end_frame]
```

Do you prefer having these changes in this PR? In that case the PR will grow a bit bigger because the changes need to be propagated to all sub-classes